cymetric/graphs.py

```diff
@@ -0,0 +1,65 @@
+import warnings
+
+import pandas as pd
+import numpy as np
+
+try:
+    from graphviz import Digraph
+    HAVE_GRAPHVIZ = True
+except ImportError:
+    HAVE_GRAPHVIZ = False
+
+try:
+    from pyne import data
+    import pyne.enrichment as enr
+    from pyne import nucname
+    HAVE_PYNE = True
+except ImportError:
+    HAVE_PYNE = False
+
+
+import cymetric as cym
+from cymetric import tools
+from cymetric.filter import get_transaction_nuc_df
+
+def get_flow_graph(evaler, senders=(), receivers=(), commodities=(), nucs=(),
+                   time=[-1, -1]):
```

Comment:
`time` should not be a list. Also it is plural, so it should be `times`.  But really, I think that it would be better to have two separate arguments, `start=None, stop=None`.  Then you check if None, rather than a potentially valid value of -1.  Also please indicate whether these times are inclusive or exclusive,